projects/packages/jetpack-mu-wpcom/tests/php/features/launchpad/class-launchpad-task-definitions-test.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Test class for Launchpad_Task_Lists.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+/**
+ * Test class for Launchpad_Task_Lists.
+ *
+ * @coversDefaultClass Launchpad_Task_Lists
+ */
+class Launchpad_Task_Definitions_Test extends \WorDBless\BaseTestCase {
+	/**
+	 * Set up.
+	 */
+	public function set_up() {
+		parent::set_up();
+
+		add_filter(
+			'wpcom_launchpad_extended_task_definitions',
+			function () {
+				return array(
+					'test_task1'            => array(),
+					'test_task2'            => array(),
+					'test_task3'            => array(),
+					'test_task_with_id_map' => array(
+						'id_map' => 'test_task_id_map',
+					),
+				);
+			}
+		);
```

Comment:
Should we clear out the `launchpad_checklist_task_statuses` option in this function between tests?
```suggestion
			delete_option( 'launchpad_checklist_tasks_statuses' );
		);
```

Reply:
We are using worddbless that clears most of the WP state after each test runs: https://github.com/Automattic/wordbless/blob/0.4.2/src/BaseTestCase.php#L29